pkg/network/master/egressip.go

```diff
@@ -163,6 +165,18 @@ func (eim *egressIPManager) poll(stop chan struct{}) {
 	}
 }
 
+func isNodeConditionGood(cond v1.NodeCondition) bool {
+	isNodeGood := true
+	if cond.Type == v1.NodeReady {
+		klog.Warningf("Node condition type is %s", cond.Type)
```

Comment:
debugging leftover? Don't log warnings about things that aren't warning-worthy...